docs/backends/BCS.md

```diff
@@ -0,0 +1,252 @@
+- `<test-access-id>` - Access ID to access Alibaba Cloud services through restful API. 
+- `<test-access-key>` - Access key to access Alibaba Cloud services through restful API. 
+
+```hocon
+backend {
+  providers {
+    BCS {
+      config {
+        # BCS related configurations mentioned above
+       
+        filesystems {
+          oss {
+            auth {
+              endpoint = "<test-oss-endpoint>"
+              access-id = "<test-access-id>"
+              access-key = "<test-access-key>"
+            }
+          }
+        }
+        
+        default-runtime-attributes {
+        # ... to be filled in
+        }
+      }
+    }
+  }
+}
+```
+
+### Runtime Attributes
+
+This backend support additional runtime attributes that are specified in the configuration key `backend.providers.BCS.config.runtime-attributes`. 
+It uses the same syntax as specifying runtime attributes in a task in WDL. A typical runtime attributes example for BCS backend looks like this:
+
+```hocon
+backend {
+  providers {
+    BCS {
+      config {
+        # BCS and OSS related configurations mentioned above
+       
+        default-runtime-attributes {
+          cluster: "OnDemand ecs.sn1ne.large "
+          mounts: "oss://<test-bucket>/inputs/ /home/inputs/ false"
+          docker: "ubuntu/latest oss://<test-bucket>/registry/ubuntu/"
+          userData: "key value"
+          reserveOnFail: true
+          autoReleaseJob: true
+          verbose: false
+          workerPath: "oss://<test-bucket>/cromwell_test/worker.tar.gz"
+          systemDisk: "cloud 50"
+          dataDisk: "cloud 250 /home/data/"
+          timeout: 3000
+        }
+      }
+    }
+  }
+}
+```
+
+#### cluster
+
+There are two different ways of specifying an Alibaba Cloud BatchCompute cluster in which workflow jobs run.
+
+- Reserved cluster - A pre-created cluster ID in BatchCompute service like this:
+
+  ```hocon
+      default-runtime-attributes {
+        cluster: "cls-your-cluster-id"
+      }
+  ```
+
+- Auto cluster - Cluster configuration to create a new runtime cluster bound to the workflow job:
+
+  - `<resource-type>` - Type of resource, can only support `OnDemand` and `Spot` currently.
+  - `<instance-type>` - Type of VM instance. Go to <a href="https://help.aliyun.com/document_detail/25378.html" target="_blank">Alibaba Cloud BatchCompute Instance Type</a> to choose a suitable type for you.
+  - `<image-id>` - Image ID of Alibaba Cloud BatchCompute service to create a VM.
+
+  ```hocon
+      default-runtime-attributes {
+        cluster: "<resource-type> <instance-type> <image-id>"
+        # Maybe like cluster: "OnDemand ecs.sn1ne.large img-ubuntu"
+      }
+  ```
+
+#### mounts
+
+BCS jobs can mount an OSS object or an OSS prefix to local filesystem as a file or a directory in VM. 
+It uses distribute-caching and lazy-load techniques to optimize concurrently read reqeust of OSS file system. 
```

Comment:
> It uses distribute-caching and lazy-load techniques to optimize concurrently read reqeust of OSS file system. 

"It uses distribute-caching and lazy-load techniques to optimize concurrently read **requests** of **the** OSS file system."